src/internal.ts

```diff
@@ -36,10 +36,7 @@ export {
 export { createVenue, CreateVenueParams } from '~/api/procedures/createVenue';
 export { inviteAccount, InviteAccountParams } from '~/api/procedures/inviteAccount';
 export { subsidizeAccount, SubsidizeAccountParams } from '~/api/procedures/subsidizeAccount';
-export {
-  issueTokens as issueAsset,
-  IssueTokensParams as IssueAssetParams,
```

Comment:
I meant that we should keep the name and remove the aliases. We decided to use "tokens" to refer to units of an asset, so, in this context, "issueTokens" is fine